server/lib/super-express.js

```diff
@@ -13,7 +13,7 @@ const defaultOptions = {
   helmet: true,
 };
 
-module.exports = (options = {}) => {
+module.exports = function (options = {}) {
```

Comment:
why not arrow function?

Reply:
server/app.js:8 we construct it, so arrow func not working.